### DISCUSSION TITLE
Add tooltips for the settings

### DISCUSSION THREAD

**Discussion by quasiblob** [bmaltais/kohya_ss#286] [Ideas]
Could it be possible to add some tooltips?

Some of the values and settings are not so obvious, what values one should try and what some option is supposed to do.

At least make some file available that could be filled by any user, if it is too much work otherwise.

I know there are already videos about this GUI, but it would be way easier to simply hover over GUI element and see what it does, instead having to watch some 1 hour video again.

**Comment by bmaltais** [comment]
Gradio does not support tooltip natively. I know aito1112 integrated them via so e king of J's and CSS... But I am not great with html so I have not been able to figure it out yet. If someone want to create a PR to add tooltip support it would be great

**Reply by quasiblob** [reply]
What a bummer - a web based UI that doesn't support CSS hover? Is that even possible. 😅

Something simple like this? Could it work?
https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_display_element_hover

I'm a total noob with Python and not that good programmer either and I've only got the basics of JS/html and some web stacks, as I'm mostly into game engines so too bad I can't myself be of much help :(

But I could help figuring out / writing tooltips (also while being a ML noob).

**Reply by bmaltais** [reply]
I am sure something like this would be doable... just a question on implementing it. It can be done in auto1111 so it must be possible.
